src/store/normalize-data.js

```diff
@@ -79,6 +79,11 @@ export const createInitialPipelineState = () => ({
     active: {},
     enabled: {},
   },
+  filters: {
```

Comment:
we have a filters section in the UI,[Nit] When I saw this, it reminded me of those filters which we have above. If possible we can rename this to pipelineSlice or slice ?

Reply:
hey i think the idea is later on we might merge those filters in the UI together with slicing function. Hence I thought it makes sense to put it here.